demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1039,7 +1059,9 @@ export class DemoMeetingApp implements
       this.meetingSession.audioVideo.setContentAudioProfile(AudioProfile.fullbandMusicMono());
     }
     this.audioVideo = this.meetingSession.audioVideo;
-
+    if (this.enableSimulcast) {
+      this.audioVideo.chooseVideoInputQuality(1280, 720, 15, 1400);
+    }
```

Comment:
Can you explain why this has snuck into this PR?

Reply:
I noticed that the selection event listener is triggered on "change", `this.audioVideo.chooseVideoInputQuality(1280, 720, 15, 1400);` is never called on simulcast. Current simulcast uplink policy only supports 720p

Reply:
You might need to unpack this for me a bit more, because that sounds like a bug independent of this change, right?

Reply:
yeah, this is a bug that is independent of this change.